apps/meteor/app/livechat/client/lib/chartHandler.ts

```diff
@@ -128,12 +129,11 @@ export const drawLineChart = async (
 	chartContext: { destroy: () => void } | undefined,
 	chartLabels: string[],
 	dataLabels: string[],
-	dataSets: number[],
+	dataSets: number[][],
```

Comment:
Same here

Reply:
Yeah, it was wrong before, but since the files that consumed this type were all in js, no error was raised